### ISSUE TITLE
Do not consider files-duplicate for SMALL files

### ISSUE THREAD

**Issue by socketpair** [rpm-software-management/rpmlint#833]
In Python RPMS:

```
$ xxd usr/lib/python3.10/site-packages/pyrad-2.4-py3.10.egg-info/zip-safe
00000000: 0a                                       .
$ xxd usr/lib/python3.10/site-packages/pyrad-2.4-py3.10.egg-info/dependency_links.txt
00000000: 0a
```
These two files both contain just one empty newline. It's not a bug to have two such files.

**Comment by marxin** [comment]
That seems reasonable to me. We can add a config value where one can define a minimum file size that should be reported as `files-duplicate`.
